trace to match functions containing a period (golang)

golang functions can contain a period, ".". Here's an example, hello.go:

```go
package main

import "fmt"

func main() {
        fmt.Println("Hello, BPF!")
}
```

Compiled with:

```Shell
# gccgo -o hello hello.go
```

Now function counting libgo with funccount, while hello is run in another terminal:

```Shell
# ./funccount.py 'go:fmt.*'
Tracing 160 functions for "go:fmt.*"... Hit Ctrl-C to end.
^C
FUNC                                    COUNT
fmt..import                                 1
fmt.padString.pN7_fmt.fmt                   1
fmt.fmt_s.pN7_fmt.fmt                       1
fmt.WriteString.pN10_fmt.buffer             1
fmt.free.pN6_fmt.pp                         1
fmt.fmtString.pN6_fmt.pp                    1
fmt.doPrint.pN6_fmt.pp                      1
fmt.init.pN7_fmt.fmt                        1
fmt.printArg.pN6_fmt.pp                     1
fmt.WriteByte.pN10_fmt.buffer               1
fmt.Println                                 1
fmt.truncate.pN7_fmt.fmt                    1
fmt.Fprintln                                1
fmt.$nested1                                1
fmt.newPrinter                              1
fmt.clearflags.pN7_fmt.fmt                  2
Detaching...
```

That's pretty awesome.

Now tracing the fmt.Println with trace:

```Shell
# ./trace.py 'go:fmt.Println'
/virtual/main.c:6:17: error: expected identifier or '('
struct probe_fmt.Println_1_data_t
                ^
/virtual/main.c:17:26: error: expected identifier or '('
BPF_PERF_OUTPUT(probe_fmt.Println_1_events);
                         ^
/virtual/include/bcc/helpers.h:69:8: note: expanded from macro 'BPF_PERF_OUTPUT'
struct _name##_table_t { \
       ^
/virtual/main.c:17:26: error: expected identifier or '('
/virtual/include/bcc/helpers.h:78:8: note: expanded from macro 'BPF_PERF_OUTPUT'
struct _name##_table_t _name
       ^
/virtual/main.c:20:14: error: expected ';' after top level declarator
int probe_fmt.Println_1(struct pt_regs *ctx)
             ^
             ;
4 errors generated.
Failed to compile BPF module 
```

CC @goldshtn

--- a/tools/trace.py
+++ b/tools/trace.py
@@ -76,6 +76,8 @@ class Probe(object):
                 self.probe_num = Probe.probe_count
                 self.probe_name = "probe_%s_%d" % \
                                 (self._display_function(), self.probe_num)
+                if self.probe_name.find(".") > 0:   # for golang
+                    self.probe_name = self.probe_name.replace(".", "_DOT_")
 
         def __str__(self):
                 return "%s:%s:%s FLT=%s ACT=%s/%s" % (self.probe_type,

